workers/data_refinery_workers/processors/illumina.R

```diff
@@ -377,8 +378,11 @@ if (length(exprColumns) == 0)
 exprData <- as.matrix(data[,exprColumns,drop=FALSE])
 rownames(exprData) <- probeIDs
 
+# Throw out NA rows
+exprData <- na.omit(exprData)
```

Comment:
I should take this out, I don't think I intended to commit this change, because in my experience, usually only a handful of columns have missing values.

I initially added this as a temporary fix for some samples that were failing with the following issue:
```
Error in if (max(x) > 100) x <- log2(x) :
  missing value where TRUE/FALSE needed
Calls: scanNorm -> doLog2
```
I will take another look at the code and see if there is an easy way to ignore NAs that occur only in single columns.

Reply:
Marking as resolved for now because we decided to open https://github.com/AlexsLemonade/refinebio/issues/2870 to track this